src/test/java/calculator/domain/NumberTest.java

```diff
@@ -0,0 +1,33 @@
+package calculator.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class NumberTest {
+
+  @ParameterizedTest
+  @CsvSource({
+      "10, PLUS, 5, 15",
+      "10, MINUS, 3, 7",
+      "4, MULTIPLY, 5, 20",
+      "10, DIVIDE, 2, 5"
+  })
+  void applyTest(int a, Operator operator, int b, int expected) {
+    calculator.domain.Number num1 = new calculator.domain.Number(a);
+    calculator.domain.Number num2 = new calculator.domain.Number(b);
+    calculator.domain.Number result = num1.apply(operator, num2);
```

Comment:
```suggestion
    Number result = new Number(a).apply(operator, new Number(b));
```
저는 간단한 테스트는 이렇게도 접근해봄